src/components/button.js

```diff
@@ -1,9 +1,11 @@
-function button(textContent,htmlClass) {
-	if (htmlClass === undefined) {
-		return `<button class="button">${textContent}</button>`;
-	}
+function button(textContent,dataValue,htmlClass) {
```

Comment:
@Willian1661 optional parameters must be after required parameters

Reply:
even if the data its more important than the other? than  the htmlClass for example

Reply:
if the parameter is optional then it's less important that a mandatory parameter!

otherwise you gonna use the function like this

```js
button('Some text', null, 'some-class')
```

Reply:
no, i'm thinking now about if someone call a button he'll stylize it and in case now we're using bulma we need that parameter, so no the class  parameter it's not a optional parameter. 🤔

Reply:
this class parameter still was a optional one, because i was thinking about still using css, but now this is past waters :p

Reply:
but if you have a data value, you need a class to identify the button
then I would say class has more priority then data-value

Reply:
yea make sense,it's ok now